src/nrncvode/cvodeobj.cpp

```diff
@@ -1442,36 +1477,49 @@ void Cvode::statistics() {
 
 void Cvode::matmeth() {
     switch (ncv_->jacobian()) {
-    case 1:
-        CVDense(mem_, neq_);
+    case 1: {
+        /* Create dense SUNMatrix for use in linear solver */
+        SUNMatrix A = SUNDenseMatrix(neq_, neq_);
+        assert(A);
+
+        /* Create dense SUNLinearSolver object for use by CVode */
+        SUNLinearSolver LS = SUNDenseLinearSolver(y_, A);
+        assert(LS);
+
+        /* attach the matrix and linear solver to CVode */
+        int flag = CVDlsSetLinearSolver(mem_, LS, A);
+        assert(flag == CVDLS_SUCCESS);
+
+        /* Set jacobian and allocate memory */
+        /* (TODO Michael Hines: is there a jacobian for dense matrix
```

Comment:
I did not review this at the time of the initial PR

Reply:
We are using the default built-in jacobian from sundials. I changed the comment to mention that.